### PR TITLE
Fix to not lose track of whether an index buffer is bound

### PR DESCRIPTION
**Pull request by Tony-LunarG**
Fixes false positive in several CTS tests

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -3952,7 +3952,7 @@ static void set_cb_pso_status(GLOBAL_CB_NODE *pCB, const PIPELINE_NODE *pPipe) {
     // Account for any dynamic state not set via this PSO
     if (!pPipe->graphicsPipelineCI.pDynamicState ||
         !pPipe->graphicsPipelineCI.pDynamicState->dynamicStateCount) { // All state is static
-        pCB->status = CBSTATUS_ALL;
+        pCB->status |= CBSTATUS_ALL;
```
Comment (chrisforbes) [inline layers/core_validation.cpp]: Looks correct. Do we want to rename `CBSTATUS_ALL` to something less misleading?

**Comment by chrisforbes** [comment]
:+1:

File: layers/core_validation.cpp (hunk 1 above)
Comment (tobine) [inline layers/core_validation.cpp]: @chrisforbes Yeah we should do something here. My thought was that we may want to keep a separate bitmask that's clearly just the dynamic state, and keep those as the lowermost bits of pCB->status. We could still track pCB->status as a single uint64_t, and there is probably more non-dynamic state that it makes to track there at some point. This fix is :+1: for the immediate problem.
I'll file separate issue for cleanup that needs to occur at some point.

**Comment by Tony-LunarG** [comment]
Cherry picked to master

**Pull request closed without merging**